app/assets/javascripts/components/rooms/room_page.es6.jsx

```diff
@@ -0,0 +1,89 @@
+class RoomPage extends Component {
+
+  // --------------------------------------------------
+  // Setup
+  // --------------------------------------------------
+  constructor(props) {
+    super(props);
+    this._listener = (state) => this.setState(state);
+  }
+
+  // --------------------------------------------------
+  // Props
+  // --------------------------------------------------
+  static get propTypes() {
+    return {
+      id: React.PropTypes.number.isRequired,
+      profile: React.PropTypes.object.isRequired,
+    };
+  }
+
+  // --------------------------------------------------
+  // Lifecycle
+  // --------------------------------------------------
+  componentWillMount() {
+    this.setState(ProfileStore.getState());
+    this.setState(RoomStore.getState());
+    this.setState(ViewStore.getState());
+  }
+
+  componentDidMount() {
+    ProfileStore.listen(this._listener);
+    RoomStore.listen(this._listener);
+    ViewStore.listen(this._listener);
+    RoomActions.fetchRoom(this.props.id);
+    ViewActions.attachListener();
+  }
+
+  componentWillUnmount() {
+    ProfileStore.unlisten(this._listener);
+    RoomStore.unlisten(this._listener);
+    ViewStore.unlisten(this._listener);
+  }
+
+  // --------------------------------------------------
+  // Helpers
+  // --------------------------------------------------
+  generateOptions() {
+    return [
+      {
+        action: () => RoomActions.toggleEditability(),
+        content: this.state.editable ? 'Finish' : 'Edit',
+      },
+    ];
+  }
+
+  selectProfile() {
+    return this.state.profile ?
+           this.state.profile :
+           this.props.profile;
+  }
+
+  // --------------------------------------------------
+  // Render
+  // --------------------------------------------------
+  render() {
+    var room = this.state.room;
+    return (
+      <div style={StyleConstants.pages.wrapper}>
+        <Header
+          active={true}
+          profile={this.selectProfile()} />
```

Comment:
A change was just made to the `Header` component, it defaults to `active=true` so you don't need to pass that in!
Can just do `<Header profile={this.selectProfile()} />`!